packages/grid/data-grid/src/DataGrid.tsx

```diff
@@ -27,6 +27,7 @@ export type DataGridProps = Omit<
   | 'licenseStatus'
   | 'options'
   | 'onRowsScrollEnd'
+  | 'onVirtualPageChange'
```

Comment:
What about naming it?

```suggestion
  | 'onViewportRowChange'
```

Per the `GRID_VIEWPORT_ROW_CHANGE` event name proposal in https://github.com/mui-org/material-ui-x/pull/1715#pullrequestreview-689902319.

Reply:
Updated